content/SCALE/SCALETutorials/SystemSettings/ManageBootEnvironSCALE.md

```diff
@@ -116,36 +116,23 @@ Enter a new name using only alphanumeric characters, and/or the allowed dashes (
 The **Source** field displays the boot environment you are cloning. If the displayed name is incorrect, close the window and select the correct boot environment to clone.
 
 Click **Save**.
-{{< /expand >}}
-
-{{< expand "Rename (Click to expand)" "v" >}}
-You can change the name of any boot environment on the **System > Boot** screen.
-Click the <span class="material-icons">more_vert</span> for a boot environment, and then select **Rename** to open the **Rename Boot Environment** window.
-
-{{< trueimage src="/images/SCALE/SystemSettings/RenameBootEnvironmentSCALE.png" alt="Rename Boot Environment" id="Rename Boot Environment" >}}
 
-Enter a new name using only alphanumeric characters, and/or the allowed dashes (-), underscores (_), and periods (.) characters.
-
-Verify the boot environment in **Name** is the one you want to rename.
+### Deleting a Boot Environment
 
-Click **Save**.
-{{< /expand >}}
-
-{{< expand "Delete (Click to expand)" "v" >}}
-Deleting a boot environment removes it from the **System > Boot** screen and from the boot menu.
+Deleting a boot environment removes it from the ** Boot Environment** screen and the boot menu.
 
 Click the <span class="material-icons">more_vert</span> for a boot environment, and then select **Delete** to open the **Delete** dialog.
 Select **Confirm** and then click **Delete**.
 
 {{< trueimage src="/images/SCALE/SystemSettings/DeleteBootEnvironmentSCALE.png" alt="Delete Boot Environment" id="Delete Boot Environment" >}}
 
 You cannot delete the default or any active entries.
-Because you cannot delete an activated boot entry, this option does not display for activated boot environments.
-To delete the active boot environment, first activate another entry and then delete the environment you want to remove.
-{{< /expand >}}
+Because you cannot delete an activated boot entry, this option does not show for activated boot environments.
+To delete the active boot environment, first, activate another entry and then delete the environment you want to remove.
+
+### Keeping a Boot Environment
 
-{{< expand "Keep/Unkeep (Click to expand)" "v" >}}
-By default, TrueNAS prunes boot environments when the **boot-pool** has no remaining storage space.
+By default, TrueNAS prunes boot environments when the boot pool runs out of remaining storage space.
 
 **Keep** toggles with the **Unkeep** option, and they determine whether the TrueNAS updater can automatically delete this boot environment if there is not enough space to proceed with an update.
```

Comment:
The more_vert icon is incorrect for this version - there's a different icon used for the Keep option.